protocol/xmrmaker/offers/offer_manager.go

```diff
@@ -127,24 +128,6 @@ func (m *Manager) TakeOffer(id types.Hash) (*types.Offer, *types.OfferExtra, err
 	return offer.offer, offer.extra, nil
 }
 
-// DeleteOfferFromDB deletes the offer from the database.
-func (m *Manager) DeleteOfferFromDB(id types.Hash) error {
-	return m.db.DeleteOffer(id)
-}
```

Comment:
It seemed unsafe that this was an exported function bypassing our read-write lock. Callers now use `DeleteOffer` which now returns any serious errors.